crates/stellar-registry-cli/src/commands/install.rs

```diff
@@ -33,70 +35,135 @@ pub enum Error {
     Io(#[from] std::io::Error),
     #[error(transparent)]
     Strkey(#[from] stellar_strkey::DecodeError),
+    #[error(transparent)]
+    Config(#[from] stellar_cli::config::locator::Error),
 }
 
 impl Cmd {
     pub async fn run(&self) -> Result<(), Error> {
-        let contract_id = testnet::contract_id();
-        let network = testnet::network();
-        let out_dir = self.out_dir.as_ref().unwrap();
-        let mut out_file = out_dir.join(&self.deployed_name);
+        // Use the network config from flattened args
+        let network_passphrase = self
+            .config
+            .network
```

Comment:
You should use `self.config.get_network()` which will resolve the network including the passphrase so it's not required that it was passed explicitly.